src/js/redux/reducers/search/filters/locationFilterFunctions.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Created by michaelbray on 12/12/16.
+ */
+
+import { Set } from 'immutable';
+
+const updateSelectedLocations = (state, value, direction) => {
+    let updatedSet = new Set(state);
+
+    if (updatedSet.includes(value) && direction === 'remove') {
+        updatedSet = updatedSet.delete(value);
+    }
+    else if (!updatedSet.includes(value) && direction === 'add') {
+        updatedSet = updatedSet.add(value);
+    }
+
+    return updatedSet;
+};
+
+export default updateSelectedLocations;
```

Comment:
Can we change this to not be a `default` export? I understand the linter will give you a warning because there is only one export in the file, but this will:

1. allow us to add more exported functions in the future without changing the `import` syntax in other files
2. be consistent with how other plain JS files that export functions behave

If you can't handle the linter warnings, you can suppress them using:
```
/* eslint-disable import/prefer-default-export */
// provide an explanation - required!
your code
/* eslint-enable import/prefer-default-export */
```
Make sure you re-enable after your code block!